polaris/setup.py

```diff
@@ -59,6 +61,10 @@ def setup_tasks(work_dir, task_list=None, numbers=None, config_file=None,
     copy_executable : bool, optional
         Whether to copy the model executable to the work directory
 
+    clean : bool, optional
+        Whether to delete the contents of the base work directory before
+        setting up tasks
```

Comment:
Do you think it's better to clean the base directory rather than only the task directories that are being set up? The latter seems preferable to me.

Reply:
Yes, that would be better.  It's also substantially more complicated to implement, but I can do that.

Reply:
If it's substantially more work, we can leave it for another time. You already have put a ton of effort into this!

Reply:
It'll be fine.  I was just lazy...

Reply:
Okay, this was not difficult to do "correctly".  It should now delete the individual task work directories and then the shared step work directories (if they aren't inside of the task work directories).  I tested this with both `polaris setup` and `polaris suite`, and it seems to work as expected

Reply:
Glad to hear it. Looks good to me by visual inspection.